tsconfig.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
this file replaces `jsconfig.json`. copy the paths from there to here and delete the other file

Reply:
noted